libs/pageserver_api/src/models.rs

```diff
@@ -368,7 +368,7 @@ pub struct TenantInfo {
 /// This represents the output of the "timeline_detail" and "timeline_list" API calls.
 #[derive(Debug, Serialize, Deserialize, Clone)]
 pub struct TimelineInfo {
-    pub tenant_id: TenantId,
+    pub tenant_id: TenantShardId,
```

Comment:
Does this need a swagger change?

Reply:
No, the values are backward & forward compatible & swagger doesn't really have a concept of a tenant ID anyway -- this will be more of a general update for folks working on control plane that the format of IDs has been extended (no new-style IDs will exist until the control plane learns to create them)